versioned_docs/version-1.2.x/surrealql/statements/define/field.mdx

```diff
@@ -80,21 +83,34 @@ DEFINE FIELD posts.* ON TABLE user TYPE record<string>;
 ```
 ### Making a field optional
 
+You can make a field optional by using the `option` type. This allows you to store `NONE` values in the field.
+
 ```surql
 -- A user may enter a biography, but it is not required.
 -- By using the option type you also allow for NONE values.
 DEFINE FIELD biography ON TABLE user TYPE option<string>;
 ```
+
+The example below shows how to define a field `user` on a `POST` table. The field is of type [record](/docs/surrealdb/surrealql/datamodel/overview). This means that the field can store a `record<user>` or `NONE`.
+
+```surql
+DEFINE FIELD user ON TABLE POST TYPE option<record<user>>;
+```
+
+
+
 ### Setting a default value
 
+You can set a default value for a field using the `DEFAULT` clause. The default value will be used if no value is provided for the field.
```

Comment:
One thought here: I wonder if we could add a hint that default can be a good alterative to `option` in some cases? e.g. something like this:

"Defaults are commonly used in a similar fashion to the `option` type in strict schemas if a field must be required, such as by returning a `""` empty string by default to represent the lack of any string data."

And then add this example from above but with "" as default instead of using an option:

DEFINE FIELD biography ON TABLE user TYPE string;
DEFAULT "";

Reply:
I reckon this might be a bit confusing because the way we recommend using `DEFAULT` is like placeholders if nothing is added to the field but `Option` is more like field can be there or not.